re-docs/highflow/highflow_sensors.hexpat

```diff
@@ -0,0 +1,11 @@
+le u16 firmware @ 0x3;
+le u16 serial_number @ 0x9;
+
+le u16 raw_pressure @ 0x11;
```

Comment:
Are these pressure values useful? Any reason for not exposing them? If they are useful, but require something else from Aquasuite that we can't do from the driver, that should be noted in the docs.

Reply:
I don't think they are useful for the high flow usb sensor. They are probably related to the mps pressure sensors.

Reply:
Then that's something we'll have to verify in #89, if we get some info from there. Thanks!